deploy/docker/math.Dockerfile

```diff
@@ -1,11 +1,8 @@
-FROM docker.io/clojure:tools-deps
+FROM --platform=linux/amd64 docker.io/clojure:tools-deps-1.11.1.1208-alpine
 
 WORKDIR /app
 COPY . .
 
-# Install clojure and fetch dependencies
-RUN clojure -A:dev -P
```

Comment:
Runs will be faster if we fetch the dependencies (all that happens when run with `-P`) during the container build phase, instead of ever time the container is run (which is what will happen if this is deleted). However, I think it's fine to exclude downloading the dev dependencies by leaving out `-A:dev` from this command.